src/api/database/models/User.ts

```diff
@@ -37,6 +38,10 @@ class User extends ParanoidModel<
   @Column(STRING)
   clientId: string;
 
+  @Index({
+    name: 'user-roles-index',
```

Comment:
Do you really need to specify the name?

Reply:
'user.roles' will be auto-generated to be the name of the index so removed.